### DISCUSSION TITLE
Search lines

### DISCUSSION THREAD

**Discussion by savchenko** [linrongbin16/fzfx.nvim#737] [Ideas]
Default `:FzfxLiveGrep` works great, but there seem to be no obvious way to replicate common "grep twice" pattern.

Consider the command: `grep -r -w -e 'import' * | grep -w 'User'`. It searches for all lines that have both "import" and "User".

Original [fzf.vim](https://github.com/junegunn/fzf.vim) has `:Lines` that does exactly that. I think it would be a great addition to the `fzfx.nvim`, maybe as a part of https://github.com/linrongbin16/fzfx.nvim/issues/523.

**Comment by linrongbin16** [comment]
You are talking about the pipe in the shell? i.e. the `|` operator in shell?

If yes, then sorry the `FzfxLiveGrep` doesn't support this now. It only supports a single `rg` command with multiple arguments.

But we could take a look see if we could support the `:Lines` from fzf.vim.

**Reply by linrongbin16** [reply]
I create a tracking issue for this feature: https://github.com/linrongbin16/fzfx.nvim/issues/749.

Close this discussion now.
